### PR TITLE
fix(dataprocessing): minute pricing precision

### PR DESCRIPTION
**Pull request by impolitepanda**
fix(dataprocessing): price and tax computation
fix(dataprocessing): memory overhead computation

Signed-off-by: Yann Pauly <yann.pauly@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `bugfix/MANAGER-5026`
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | 
| License          | BSD 3-Clause

## Description

This PR adds three new parameters to the pricing component allowing better control on the way price is displayed (disabling rounding and control of the number of decimals displayed)
It also fixes a few bugs on price calculation.

### CODE REVIEW

File: packages/manager/modules/catalog-price/src/catalog-price.controller.js [hunk 1]
```diff
@@ -38,16 +42,66 @@ export default class {
...
+   */
   getPriceText(priceInCents) {
     const locale = this.TranslateService.getUserLocale().replace('_', '-');
-    const price = this.getIntervalPrice(priceInCents / 100000000);
+    let price;
+    if (this.performRounding) {
+      price = this.getIntervalPrice(priceInCents / 100000000);
+    } else {
+      price = this.getIntervalPriceWithoutRounding(priceInCents / 100000000);
+    }
```
Comment (JeremyDec) [inline packages/manager/modules/catalog-price/src/catalog-price.controller.js:57]: **suggestion:** for simplicty
```suggestion
    let price = priceInCents / 100000000;
    price = this.performRounding ? this.getIntervalPrice(price) : this.getIntervalPriceWithoutRounding(price)
```

File: packages/manager/modules/catalog-price/src/catalog-price.controller.js [hunk 2]
```diff
@@ -38,16 +42,66 @@ export default class {
...
-    }).format(price);
+    };
+
+    if (!isUndefined(this.minimumFractionDigits)) {
+      numberFormatOptions.minimumFractionDigits = this.minimumFractionDigits;
+    }
+
+    if (!isUndefined(this.maximumFractionDigits)) {
+      numberFormatOptions.maximumFractionDigits = this.maximumFractionDigits;
+    }
```
Comment (JeremyDec) [inline packages/manager/modules/catalog-price/src/catalog-price.controller.js:70]: **praise:** Nice catch :)

File: packages/manager/modules/catalog-price/src/catalog-price.controller.js (hunk 1 above)
Comment (impolitepanda) [reply packages/manager/modules/catalog-price/src/catalog-price.controller.js:57]: Hi ! Will apply while separating the comits :)

**Comment by impolitepanda** [comment]
> Hello @impolitepanda, thanks for your PR!
> 
> Can you split the changes to `catalog-price` component in to a separate commit please ?

Doing :)

**Comment by impolitepanda** [comment]
> > Hello @impolitepanda, thanks for your PR!
> > Can you split the changes to `catalog-price` component in to a separate commit please ?
> 
> Doing :)

@JeremyDec And done :) Your comment has been applied on the if/else refacto as well

**Pull request merged**